dev/Common/Converters.idl

```diff
@@ -0,0 +1,14 @@
+﻿namespace MU_XC_NAMESPACE
+{
+
+[bindable]
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+runtimeclass CornerRadiusFilterConverter : Windows.UI.Xaml.Data.IValueConverter
+{
+    CornerRadiusFilterConverter();
+    Object Convert(Object value, Windows.UI.Xaml.Interop.TypeName targetType, Object parameter, String language);
```

Comment:
Since this implements the interface, do we need the methods declared here? If they're coming from the interface I wouldn't think so.

Reply:
Looks like we have to keep them otherwise I get compile errors :\
`error : Type 'Microsoft.UI.Xaml.Controls.CornerRadiusFilterConverter' does not have a default interface`

Reply:
Ahhh yeah, MIDL is fun. You need to add [default_interface] attribute the the type. You can search the code base for some other examples.

Reply:
yeah that worked👍